ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python3
+import argparse
+import os
+import subprocess
```

Comment:
Please use Executable from wxflow.

Reply:
`subprocess` removed in favor of `Executable`.  Done at [`2510b32`](https://github.com/NOAA-EMC/GDASApp/commit/2510b32a33a7b5060a227a83cebe002e797acdd6)